azure-pipelines.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       - script: echo '##vso[task.prependpath]$(HOME)/.deno/bin/'
       - script: eslint **/*.ts --max-warnings=0
       - script: deno run --allow-run --allow-write --allow-read format.ts --check
-      - script: deno run --allow-run --allow-net --allow-write --allow-read test.ts
+      - script: deno run --allow-run --allow-net --allow-write --allow-read --config=tsconfig.test.json test.ts
```

Comment:
We need to de-dup these commands at some point...

Probably we should try to copy [how Tokio uses Azure Pipelines ](https://github.com/tokio-rs/tokio/tree/84d5a7f5a0660dbe1e2be0e831e715f13cb59a2b/ci)

Reply:
Definitely, I'll do that in followup PR